test/eredis_sentinel_SUITE.erl

```diff
@@ -97,7 +97,7 @@ t_connection_failure_during_start_reconnect(Config) when is_list(Config) ->
     IsDead = receive {'EXIT', C, _} -> died
              after 400 -> still_alive end,
     ?assertEqual(still_alive, IsDead),
-    ?assertMatch(ok, eredis:stop(C)),
+    ?assertEqual(ok, eredis:stop(C)),
```

Comment:
If dialyzer complains that 'ok' is the only possible value in the assert, then we can just do it like this:

```suggestion
    ok = eredis:stop(C),
```

Reply:
Good suggestion, will change